README.md

```diff
@@ -21,6 +21,9 @@ On Fedora:
 $ dnf install -y golang-bin yarnpkg maven rubygem-bundler npm
 ```
 
+
+You also need rbenv installed. Any installed ruby versions will be used as backups from starting with system ruby (if available), and then from newest to oldest.
```

Comment:
So as not to break things for users, `deplist` should keep working without `rbenv` installed, and only use it if is available.

Reply:
```suggestion
Also supports `rbenv`. Any ruby versions installed with `rbenv` will be used as backups from starting with system ruby (if available), and then from newest to oldest.
```

Reply:
Tested it out, and works both with and without rbenv for me, awesome!